File: app/src/main/java/com/example/applogin/MainActivity.java
package com.example.applogin;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    private EditText usernameInput, passwordInput;
    private Button loginButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Ánh xạ các thành phần từ giao diện
        usernameInput = findViewById(R.id.username_input);
        passwordInput = findViewById(R.id.password_input);
        loginButton = findViewById(R.id.login_btn);

        // Xử lý sự kiện khi bấm nút Login
        loginButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                // Lấy dữ liệu từ EditText
                String username = usernameInput.getText().toString().trim();
                String password = passwordInput.getText().toString().trim();

                // Kiểm tra thông tin đăng nhập
                if (username.equals("admin") && password.equals("123456")) {
                    // Thông báo đăng nhập thành công
                    Toast.makeText(MainActivity.this, "Login successful!", Toast.LENGTH_SHORT).show();
                } else {
                    // Thông báo đăng nhập thất bại
                    Toast.makeText(MainActivity.this, "Invalid username or password!", Toast.LENGTH_SHORT).show();
                }
            }
        });
    }
}

File: app/src/main/java/com/example/loginapp/MainActivity.kt
package com.example.loginapp

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {

    lateinit var usernameInput : EditText
    lateinit var passwordInput : EditText
    lateinit var loginBtn : Button





    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
            usernameInput = findViewById(R.id.username_input)
            passwordInput = findViewById(R.id.password_input)

            loginBtn = findViewById(R.id.login_btn)

        loginBtn.setOnClickListener{
            var username =  usernameInput.text.toString()
            var password =  passwordInput.text.toString()
            Log.i("Test Credentials", "Username : $username and Password : $password")
        }
    }
}